extensions/scarb-doc/Cargo.toml

```diff
@@ -35,3 +36,4 @@ smol_str.workspace = true
 assert_fs.workspace = true
 scarb-test-support = { path = "../../utils/scarb-test-support" }
 walkdir.workspace = true
+multizip = "0.1.0"
```

Comment:
why not https://docs.rs/itertools/latest/itertools/fn.multizip.html that we already have pulled-in?